src/main/kotlin/skytils/skytilsmod/listeners/ChatListener.kt

```diff
@@ -219,8 +218,8 @@ class ChatListener {
                 val lines = (chatGui as AccessorGuiNewChat).chatLines
                 val drawnLines = (chatGui as AccessorGuiNewChat).drawnChatLines
                 var i = 0
-                delay(Duration.microseconds(100))
-                while (!StringUtils.stripControlCodes(lines[i].chatComponent.unformattedText)
+                delay(100)
```

Comment:
Isn't delay defaulted to milliseconds instead of microseconds

Reply:
yeah I wanted it to be slightly less than a millisecond so it'd clear the lines faster

Reply:
ah yeah, woops i'll fix that now